include/deal.II/base/tensor.h

```diff
@@ -1849,41 +1919,41 @@ invert (const Tensor<2,dim,Number> &t)
   switch (dim)
     {
     case 1:
-      return_tensor[0][0] = 1.0/t[0][0];
+      return_tensor[0][0] = 1.0/t[unchecked_index(0U)][unchecked_index(0U)];
```

Comment:
I tried that when I mechanically made all my changes but, lo and behold, it doesn't compile. (You'll see why if you look at line 1848 where `return_tensor` is declared :-)